### ISSUE TITLE
linear(): argument 'input' (position 1) must be Tensor, not str

### ISSUE THREAD

**Issue by chrischang5** [prateekjoshi565/Fine-Tuning-BERT#10]
![image](https://user-images.githubusercontent.com/64388914/113469150-31b83a00-9400-11eb-861f-983ffdb500d6.png)

Tried running the code but got this error on both my dataset and the example dataset.

**Comment by swarupe** [comment]
Hi @chrischang5 Even I am facing the same issue. Were you able to fix it?

**Comment by chrischang5** [comment]
Hi @swarupe, unfortunately, I was not able to fix it. I've seen talks on using older versions of Huggingface transformers but I haven't been able to test them fully.

**Comment by teejay13** [comment]
getting exact error output Any Solution

**Comment by claudio-toledo** [comment]
Same error. I was able to get it resolved to downgrade Transformers: pip install transformers==3

**Comment by alankar66** [comment]
In the BERT_Arch(nn.Module):  class change the line 

from:
 _, cls_hs = self.bert(sent_id, attention_mask=mask)

to this:-
_, cls_hs = self.bert(sent_id, attention_mask=mask, return_dict=False)

**Comment by mukherjeekoushiklearning** [comment]
Thank you so much for this solution. I was really struggling to solve this issue.
